express/features/support/steps/mobile-verification-resend-steps.ts

```diff
@@ -0,0 +1,48 @@
+
+   
+import {Given, When} from "@cucumber/cucumber";
+
+Given('that the user is on the `Check your phone` page', async function () {
+  await this.goToPath('/create/get-email');
+  await this.page.type("#email", 'tessa.ting@foo-bar.gov.uk');
+  await this.page.waitForSelector('#submit');
+  await this.page.click('#submit')
+  await this.goToPath('/create/check-email');
```

Comment:
We should generally only use `gotToPath` once in each feature.  The step code is code that automates browser interactions so we want to go to a page, click some buttons or links or fill in a form and submit it and test the final behaviour.  If we go to the page we expect a form submission to take us to, we're not really testing whether the form takes us where we expect when we submit it.

I'll guess that at later stages you might run into problems because some things that are expected to be in the session are missing.  You can incorporate the changes in the branch `BAU-stub_login` that I created this morning for Boris and change the email address used to register to `registering-successfully@gds.gov.uk` which should provide the necessary responses in the right place.

This set up code should just emulate what a user would do to get to the screen we're testing.